deploy-templates/templates/keycloak/realmrolebatch.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.sso.enabled -}}
+{{- if and .Values.sso.enabled (not .Values.sso.addonsExtensionsOIDC) -}}
```

Comment:
do we need this? the feeling that we are not creating this resource in addons?

Reply:
done